hieradata/common.yaml

```diff
@@ -513,6 +513,8 @@ ccs_hcu::pkgurl: "%{lookup('profile::ccs::common::pkgurl')}"
 ccs_monit::pkgurl: "%{lookup('profile::ccs::common::pkgurl')}"
 
 systemd::manage_udevd: true
+systemd::udev_log: 'debug'
```

Comment:
We definitely don't want to enable debug on all hosts.

Reply:
Copy i will remove it from common and just move it to a role level for the hosts he is just working with which are the ones using the fiberspec role.